### DISCUSSION TITLE
hicpro to cool

### DISCUSSION THREAD

**Discussion by ratheraarif** [open2c/cooler#449] [Q&A] [answered]
is there a way to convert allValidPairs file from hicpro pipeline to cool format. I can see that `cooler cload pairs` expects pairs file, but I have  `allValidPairs `

Thank you

**Comment by nvictus** [comment, marked as answer]
`allValidPairs` is a pairs file. Just double check the columns and provide the coordinate field numbers via the `-c1`, `-p1`, `-c2`, `-p2` options.
